cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -753,7 +752,6 @@ Slice::Python::ImportVisitor::visitDictionary(const DictionaryPtr& p)
 void
 Slice::Python::ImportVisitor::visitEnum(const EnumPtr& p)
 {
-    // TODO if a value is initialized with a constant, we need to import the type of the constant.
```

Comment:
The TODO was bogus, the generated code uses the constant value not the constant, there is no need to import the constant.